package.json

```diff
@@ -6,38 +6,50 @@
     "type": "git",
     "url": "https://github.com/MetaMask/eslint-config.git"
   },
+  "type": "module",
   "workspaces": [
     "packages/*"
   ],
   "scripts": {
-    "generate": "node ./scripts/generate-configs.js",
+    "generate": "node scripts/generate-configs.mjs",
     "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:config-validation",
     "lint:changelogs": "yarn workspaces foreach --parallel --verbose run lint:changelog",
-    "lint:config-validation": "node ./scripts/validate-configs.js",
-    "lint:eslint": "yarn eslint . --ext ts,js",
+    "lint:config-validation": "node scripts/validate-configs.mjs",
+    "lint:eslint": "eslint",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn lint:config-validation --write",
     "lint:misc": "prettier '**/*.json' '!**/rules-snapshot.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore",
     "test": "vitest"
   },
   "devDependencies": {
-    "@eslint/eslintrc": "^3.0.2",
-    "@eslint/js": "^8.57.0",
+    "@babel/core": "^7.25.2",
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-syntax-import-attributes": "^7.25.6",
+    "@eslint/config-array": "^0.18.0",
+    "@eslint/js": "^9.11.0",
+    "@jest/globals": "^29.7.0",
     "@lavamoat/allow-scripts": "^3.0.4",
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/eslint-config": "^13.0.0",
+    "@metamask/eslint-config-jest": "workspace:^",
     "@metamask/eslint-config-nodejs": "^13.0.0",
+    "@metamask/eslint-config-typescript": "workspace:^",
     "@metamask/utils": "^9.1.0",
-    "eslint": "^8.57.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import-x": "^0.5.1",
-    "eslint-plugin-jest": "^27.9.0",
-    "eslint-plugin-jsdoc": "^47.0.2",
-    "eslint-plugin-n": "^16.6.2",
-    "eslint-plugin-prettier": "^4.2.1",
+    "@types/eslint__js": "^8.42.3",
+    "@types/jest": "^29.5.12",
```

Comment:
Is this still necessary after the shift to vitest?

```suggestion
```

Reply:
It's not, good catch!